config/site/examples/music/queries/filtering/EmptyFilters.graphql

```diff
@@ -1,4 +1,4 @@
-query IgnoredFilters {
+query NullOrEmptyFilters {
```

Comment:
```suggestion
query EmptyFilters {
```

...to match the filename.

Reply:
Didn't realize you had auto-merge on (which is fine), so this was merged when I approved. I've got a followup PR to fix this here: https://github.com/block/elasticgraph/pull/41 if you want to review it.

Reply:
Approved, thanks!